### PR TITLE
Update 108-barnsley-fern.md

### PR DESCRIPTION
**Pull request by Suji04**
https://suji04.github.io/fractal-ferns/

I made a version which includes various types of fractal ferns. 4 types of ferns are shown here.

### CODE REVIEW

File: _CodingChallenges/108-barnsley-fern.md [hunk 1]
```diff
@@ -21,7 +21,13 @@ contributions:
       name: "Addo White"
     video_id: "3Q5tAWkAEBc"
     source: "https://github.com/addowhite/Console-Barnsley-Fern"
----
+  - title: "Various Fractal Ferns"
+  author:
+    name: "Sujan Dutta"
+    url: "https://github.com/Suji04"
+  url: "https://suji04.github.io/fractal-ferns/"
+
```
Comment (nilsweb) [inline _CodingChallenges/108-barnsley-fern.md]: This line contains some whitespace. Please remove it and then we can merge! 🎉

File: _CodingChallenges/108-barnsley-fern.md [hunk 2]
```diff
@@ -21,7 +21,13 @@ contributions:
       name: "Addo White"
     video_id: "3Q5tAWkAEBc"
     source: "https://github.com/addowhite/Console-Barnsley-Fern"
----
+  - title: "Various Fractal Ferns"
+  author:
```
Comment (nilsweb) [inline _CodingChallenges/108-barnsley-fern.md]: YAML is very picky about indentation. This line needs to be indented by two spaces!

File: _CodingChallenges/108-barnsley-fern.md [hunk 3]
```diff
@@ -21,7 +21,13 @@ contributions:
...
+    name: "Sujan Dutta"
```
Comment (nilsweb) [inline _CodingChallenges/108-barnsley-fern.md]: YAML is very picky about indentation. This line needs to be indented by two spaces!

File: _CodingChallenges/108-barnsley-fern.md [hunk 4]
```diff
@@ -21,7 +21,13 @@ contributions:
...
+    url: "https://github.com/Suji04"
```
Comment (nilsweb) [inline _CodingChallenges/108-barnsley-fern.md]: YAML is very picky about indentation. This line needs to be indented by two spaces!

File: _CodingChallenges/108-barnsley-fern.md [hunk 5]
```diff
@@ -21,7 +21,13 @@ contributions:
...
+  url: "https://suji04.github.io/fractal-ferns/"
```
Comment (nilsweb) [inline _CodingChallenges/108-barnsley-fern.md]: YAML is very picky about indentation. This line needs to be indented by two spaces!

File: _CodingChallenges/108-barnsley-fern.md [hunk 6]
```diff
@@ -21,7 +21,13 @@ contributions:
...
+  
+  ---
```
Comment (nilsweb) [inline _CodingChallenges/108-barnsley-fern.md]: There shouldn't be any leading whitespace here!

**Comment by gruselhaus** [comment]
@Suji04 Can you give me your OK to solve the conflicts and merge please. 🎉

**Comment by nilsweb** [comment]
Thank you for sharing! :tada:

**Pull request merged**